crypto/blake2/TODO

```diff
@@ -0,0 +1,4 @@
+The code uses NATIVE_LITTLE_ENDIAN in #if statements.  Is this right?
+
+The code used #pragma pack, which I removed.  The layout for the param structs
+is required to be fully packed, but I think it will be on all platforms.
```

Comment:
I'm not sure that's actually going to be the case.  Could you add a test for it checking the size of the struct?